pkg/operator/ceph/object/status_test.go

```diff
@@ -32,8 +32,12 @@ func TestBuildStatusInfo(t *testing.T) {
 		},
 	}
 	cephObjectStore.Spec.Gateway.Port = 80
+	cephObjectStore.Spec.Gateway.SecurePort = 443
```

Comment:
You need to test:

1. Gateway.Port exists and Gateway.SecurePort is empty, in this case only `statusInfo["endpoint"]` exists
2. Gateway.Port is empty and Gateway.SecurePort exists, in this case only `statusInfo["endpoint"]` exists
3. Gateway.Port exists and Gateway.SecurePort exists, in this case both `statusInfo["endpoint"]`and statusInfo["secureEndpoint"] exists

Reply:
Done. Thanks for the suggestion, I thought having both `port` and `securePort` will cover other conditions.